src/components/pages/NotFoundPage.tsx

```diff
@@ -0,0 +1,36 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faBiohazard } from "@fortawesome/free-solid-svg-icons";
+import { Button } from 'semantic-ui-react'
+import React, { useContext } from "react";
+import { AppContext } from "../../context";
+import { useMediaQuery } from "react-responsive";
+import { mobileDeviceOrTabletWidth } from "../../constants";
+import Translate from "../../utils/translate/translateService";
+import "./static.css";
+import { useHistory } from 'react-router-dom';
+import BrokenAb from '../../assets/images/BrokenAb.gif'
+
+export default function NotFoundPage(){
+  const isMobileDeviceOrTablet = useMediaQuery({
+    maxWidth: mobileDeviceOrTabletWidth,
+  });
+
+  const history = useHistory();
+
+  const [{ language }, _] = useContext(AppContext);
+
+  return (
+    <div className="page col-12">
+      <div className="center-container">
+        <img src={BrokenAb} height={"30%"} alt="" />
+        <p className="not-found-subheader">{Translate("NotFound", ["PartOne"])}</p>
+        <p>{Translate("NotFound", ["PartTwo"])}</p>
+        <Button 
+          className="not-found-button"
+          onClick={() => history.push(`/${language}/Explore`)}>
```

Comment:
Just for my understanding - what's the difference between `history.push()` and having a direct link to `/Explore` here?

Reply:
React router dom uses the history object to manage session state (what routes users have visited). The `history.push()` method will allow us to route while updating this session state. It's not super necessary but best practice when using react router dom, and we may want to use features provided by the history object in the future